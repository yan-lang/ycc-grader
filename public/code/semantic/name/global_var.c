int a = 10;
int c;
int d = 9*10+0;
float f;
float e;
float abc = 10.2;
float abe = 20.5+13.222+0.8;