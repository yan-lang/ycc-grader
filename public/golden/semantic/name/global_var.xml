<nameResolve>
    <def>
        <line>1</line>
        <type>var</type>
        <name>a</name>
    </def>
    <def>
        <line>2</line>
        <type>var</type>
        <name>c</name>
    </def>
    <def>
        <line>3</line>
        <type>var</type>
        <name>d</name>
    </def>
    <def>
        <line>4</line>
        <type>var</type>
        <name>f</name>
    </def>
    <def>
        <line>5</line>
        <type>var</type>
        <name>e</name>
    </def>
    <def>
        <line>6</line>
        <type>var</type>
        <name>abc</name>
    </def>
    <def>
        <line>7</line>
        <type>var</type>
        <name>abe</name>
    </def>
</nameResolve>